src/Tribe/Template_Factory.php

```diff
@@ -538,6 +469,86 @@ public function comments_off( $option_value, $option_name ) {
 		}
 
 		return $option_value;
+	}
+
+	/************************
+	 *                      *
+	 *  Deprecated Methods  *
+	 *                      *
+	 ************************/
 
+	/**
+	 * Asset calls for vendor packages
+	 *
+	 * @deprecated TBD
+	 *
+	 * @param string $name
+	 * @param array  $deps Dependents
+	 */
+	public static function asset_package( $name, $deps = array() ) {
+
+		$common = Tribe__Events__Main::instance();
+		$prefix = 'tribe-events'; // Tribe__Events__Main::POSTTYPE;
+
+		// setup plugin resources & 3rd party vendor urls
+		$vendor_url = trailingslashit( $common->plugin_url ) . 'vendor/';
+
+		self::handle_asset_package_request( $name, $deps, $vendor_url, $prefix, $common );
+	}
+
+	/**
+	 * Handles an asset package request.
+	 *
+	 * @deprecated TBD
+	 *
+	 * @param string      $name       The asset name in the `hyphen-separated-format`
+	 * @param array       $deps       An array of dependency handles
+	 * @param string      $vendor_url URL to vendor scripts and styles dir
+	 * @param string      $prefix     MT script and style prefix
+	 * @param Tribe__Main $tec        An instance of the main plugin class
+	 */
+	protected static function handle_asset_package_request( $name, $deps, $vendor_url, $prefix, $tec ) {
+
+		$asset = self::get_asset_factory_instance( $name );
+
+		self::prepare_asset_package_request( $asset, $name, $deps, $vendor_url, $prefix, $tec );
+	}
+
+	/**
+	 * Retrieves the appropriate asset factory instance
+	 *
+	 * @deprecated TBD
+	 */
+	protected static function get_asset_factory_instance( $name ) {
+		$asset = Tribe__Events__Asset__Factory::instance()->make_for_name( $name );
```

Comment:
```
return Tribe__Events__Asset__Factory::instance()->make_for_name( $name );
```

Reply:
It's deprecated code.

Reply:
does using `_deprecated_function()` make sense here and for these other deprecated asset methods?